### PR TITLE
[12] proposed edits for day one f2f morning session

### LINKED ISSUE #12: Proposed edits for day one face-to-face to Change Proposal -- morning session
Process suggestion: Use GitHub Issues for suggesting proposed changes and tracking work on proposed changes

Proposed edits:

High Level Overview table

* DPAC -- Add mention of pre-configured filters
* IHE wide glossary -- contact Mary for latest IHE-DEV glossary. Don't use the word glossary -- proposed name changed Appendix D -- Definitions

7 PCIM --- remove acute-care

* Diagram -- add reference to 7.1.1.4 under MEM DMC annotation ??? 

7.1.1.1  Change: 

* first sentence: ... device is associated or disassociated with a specific patient, or an update to an existing association.
* last sentence: ..., the report observation status field shall be marked final, ...

 7.1.1.3 

* Change to: The actor initially receives current association status followed by updates in real-time.

7.1.1.4

Change title to: Device Registration

* In first sentence state:  
  The list of medical devices that can be associated with the patient may be pre-configured or automated with MEM DMC DMIR.
* Clarify second paragraph, first sentence. 
  Change to: The IHE DMC DMIR may facilitate automated device registration.
* Examples for DMIR.  model, manufacturer, serial number, end point (IP address, port)

7.2

*  First sentence: 
   Reverse DPAM and DPAC filtering references

7.2.1

* Table 7.2.1 remove reference for reporter.
* Paragraph: Reverse DPAC and DPAM order in sentence.

7.4

* In use case example titles, change Query to Filter
* Description section before Flow section

7.4.2.1.1

* Title change to: Process Flow
* Second sentence:  The --> This

7.4.2.1.3

* typo: device identify to device identity

7.4.2.3.1

* reverse position of DPAC and DPAM

7.4.2.4

* Drop this use case

7.7

* add 7.7.1 with existing text with expectation of added out of scope examples

### PR DESCRIPTION
**Pull request by reh-imt**
Updated nn-6 asciidoc with changes proposed in the morning session of the May 23 Face-to-Face meeting in Alexandria VA.

Resolves #12

### CODE REVIEW

File: CPs/cp_nn6.adoc [hunk 1]
```diff
@@ -437,72 +459,83 @@ A Device-Patient Association Consumer that supports this option shall formulate
...
+A Device-Patient Association Consumer may query a Device-Patient Association Manager for a list of devices associated with a particular patient at present, or at a designated time in the past, or more generally for a snapshot of the Device-Patient Association map. 
 
 [.text-left]
 [underline]#Proposed Text:#
 [.text-left]
-A device (or another system) may require the identity of the patient it is connected to, for display or other purposes, but not have this information available to it, so the profile provides for a Device-Patient Association Consumer to receive reports from the Device-Patient Association Manager that provide this information for the devices it is configured to receive them for.
+A Device-Patient Association Manager may filter association messages to a Device-Patient Association Consumer for the list of devices associated with particular patients at present. Retrospective queries are currently out of scope.
```
Comment (eldonmetz) [inline CPs/cp_nn6.adoc:516]: add "and ongoing" after "at present".

File: CPs/cp_nn6.adoc [hunk 2]
```diff
@@ -437,72 +459,83 @@ A Device-Patient Association Consumer that supports this option shall formulate
...
 [underline]#Existing Text:#
 [.text-left]
-The identity of the patient associated with a device (or the lack of an associated patient identity) may be queried for. 
+For status display or for error-checking and diagnostic purposes, the Device-Patient Association Manager can respond to a targeted query by sending a query response message. 
 
 [.text-left]
 [underline]#Proposed Text:#
 [.text-left]
-For status display or for error-checking and diagnostic purposes, the Device-Patient Association Manager sends the Device-Patient Association Consumer the present association records for each device it is configured to receive.
+For status display or for error-checking and diagnostic purposes, the Device-Patient Association Manager sends the Device-Patient Association Consumer the present association records for each patient it is configured to receive.
```
Comment (eldonmetz) [inline CPs/cp_nn6.adoc:532]: Change "present" to "current".

File: CPs/cp_nn6.adoc (hunk 1 above)
Comment (eldonmetz) [reply CPs/cp_nn6.adoc:516]: Consider changing "present" to "current".

File: CPs/cp_nn6.adoc [hunk 3]
```diff
@@ -333,11 +355,11 @@ Where it is a system, it may be a comprehensive device inventory system, a “ga
 [underline]#Proposed Text:#
 
 [.text-left]
-The IHE MEM DMC DMIR may automate device registration. Device registration may also be manually accomplished during system setup and maintenance.
+The list of medical devices that can be associated with the patient may be preconfigured or automated with MEM DMC DMIR. Device registration may also be manually accomplished during system setup and maintenance. Examples of information available from MEM DMC are the device model, manufacturer, serial number, and network end point (ip address, port).
```
Comment (eldonmetz) [inline CPs/cp_nn6.adoc:358]: Remove "DMIR". Not needed, just reference profile.

**Review by eldonmetz — CHANGES_REQUESTED** [review]
Please modify per the review comments.

**Comment by eldonmetz** [comment]
@bilhar97 and @TomKowal. I applied, committed and pushed the recommended changes. Probably easiest to review by focusing in and clicking on the changes related to my morning commits. @reh-imt, thanks for this initial push, I think we are close to merging this in.

**Review by bilhar97 — APPROVED** [review]
Reviewed latest changes to cp_nn6.adoc. I did not review HTML (except index.html) since it is generated from the adoc. Changes look good to me. Approved.

**Comment by eldonmetz** [comment]
> Reviewed latest changes to cp_nn6.adoc. I did not review HTML (except index.html) since it is generated from the adoc. Changes look good to me. Approved.

Thanks, updated html and pushed that commit.

**Pull request merged**